torch/_inductor/ir.py

```diff
@@ -4489,9 +4489,12 @@ def __init__(self, *, kernel_idx, grid, kernel_args):
             arg for arg in kernel.arg_names if arg in kernel_args
         ]
 
-        mark_node_as_mutating(
-            self, *[a for a in kernel_args.values() if isinstance(a, TensorBox)]
-        )
+        mutable_args = [
+            arg
+            for key, arg in kernel_args.items()
```

Comment:
iterate over `tensors_to_clone` since key from that is guaranteed to be in kernel_args